pokemon_search/src/App.js

```diff
@@ -54,7 +54,9 @@ export default function App() {
             </li>
           ))}
       </ul>
-      <button className="btn fourth">Next 50?</button>
+      <button onClick={() => setOffset(offset + 50)} className="btn fourth">
```

Comment:
Clicking this button currently doesn't feel like new data is loading for the user. :)
A common approach is to hide the old data while new data is being loaded in.